assets/src/scripts/components/graph/state/well-log.js

```diff
@@ -141,7 +141,7 @@ export const getWellWaterLevel = memoize((opts, chartType) => createSelector(
         return {
             x: xRange[0],
             y: top,
-            width: xRange[1],
+            width: xRange[1] - xRange[0],
```

Comment:
Consider updating the test for getWellWaterLevel to test that the width is calculated correctly.

Reply:
Thanks @mbucknell - pushed new test to make sure width is calculated properly.